management/azure_mgmt_resources/spec/2017-05-10/deployment_spec.rb

```diff
@@ -31,7 +31,7 @@
     expect{@client.create_or_update(nil, 'bar', @resource_helper.build_deployment_params)}.to raise_error(ArgumentError)
     expect{@client.create_or_update('foo', nil, @resource_helper.build_deployment_params)}.to raise_error(ArgumentError)
     expect{@client.create_or_update('foo', 'bar', nil)}.to raise_error(ArgumentError)
-    expect{@client.create_or_update('~`123', 'bar', @resource_helper.build_deployment_params).value!}.to raise_error(MsRestAzure::AzureOperationError)
+    expect{@client.create_or_update('~`123', 'bar', @resource_helper.build_deployment_params).value!}.to raise_error(ArgumentError)
```

Comment:
why are we seeing this issue now? if we're not updating resources related files? shouldn't CI be failing before for this?